config/owasp/suppressions.xml

```diff
@@ -7,5 +7,26 @@
     </notes>
     <cve>CVE-2018-1258</cve>
   </suppress>
- <!-- Supression Group ends here -->
+
```

Comment:
Why we are adding so many CVEs. We only have <cve>CVE-2018-1258</cve> at the moment in master

Reply:
These CVEs appear as part of OIDC and they are false positives. These CVEs appear in their latest version as well